Server/Plugins/APIDump/Classes/World.lua

```diff
@@ -389,6 +421,30 @@ return
 						IsOptional = true,
 					},
 				},
+				Notes = "Sends the specified effect to all players in this world, except the optional ExceptClient. <b>OBSOLETE</b>, use the vector overload instead",
+			},
+			BroadcastSoundParticleEffect =
+			{
+				Params =
+				{
+					{
+						Name = "EffectID",
+						Type = "number",
+					},
+					{
+						Name = "SourcePos",
+						Type = "Vector"
```

Comment:
This is the one causing failed CI, needs to be `Vector3f` instead.

Reply:
Thanks, fixed now.